src/app/service/post.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+
+interface Post {
+  postId: number;
+  username: string;
+  valueOfLikes: number;
+  nrComments: number;
+  voteStatus: 'upvoted' | 'downvoted' | 'undefined';
+  content: string;
+  favorited: boolean;
+  imgLink: string | null;
+}
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PostService {
+  private apiUrl = 'http://localhost:8080/posts';
```

Comment:
would still this link (http://localhost:8080) still work when deployed?

Reply:
Nope, but Alex knows about this so it is not a problem